.env_sample

```diff
@@ -1,5 +1,5 @@
-export DB_NAME='dojo'
-export DB_USER='postgres'
-export DB_PASS='postgress'
-export DB_HOST='localhost'
-export DB_PORT='5432'
\ No newline at end of file
+export DB_NAME=''
```

Comment:
Its good that you put this back

Reply:
Thanks, @JamesMudidi, but not every user who wants to try out the app on will have the same database name on their system.